subsys/bluetooth/controller/ll_sw/ull_adv.c

```diff
@@ -1506,6 +1507,13 @@ int ull_adv_reset_finalize(void)
 	uint8_t handle;
 	int err;
 
+#if defined(CONFIG_BT_CTLR_ADV_PERIODIC)
+	err = ull_adv_sync_reset_finalize();
+	if (err) {
+		return err;
+	}
+#endif /* CONFIG_BT_CTLR_ADV_PERIODIC */
+
```

Comment:
`_finalise()` is not required, as lll_adv_data_reset for periodic advertising is called on every `ll_adv_sync_param_set` call.

```suggestion
```

Reply:
It seems to me that it is required to clear `adv->lll.sync`.

Reply:
The code is left here.